_posts/2015-10-02-search-widgets.md

```diff
@@ -101,6 +107,16 @@ The widget enables your customers to do a quick search for search for both a zip
 </div>
 </div>
 
+###International Office Locations Widget
+
```

Comment:
All the other widgets have some text describing what they do. You're not going to add that for this one?

Otherwise, these changes look good. Go ahead and merge :+1: